common/checker.go

```diff
@@ -0,0 +1,71 @@
+// Copyright 2016-2017 Authors of Cilium
```

Comment:
Move this to `pkg/...`. the `common` pkg is going away

Reply:
Does `pkg/common/checker.go` seem fine to you? I don't really see another good place for common infrastructure bits.